tests/test_mcp_server.py

```
@@ -1524,26 +951,192 @@ def test_serialization_edge_cases(
+                filter_instance = MockFilter.return_value
+                if filter_type == "limit_normal":
+                    filter_checker(filter_instance).assert_called_once_with(5)
+                elif filter_type == "limit_max":
+                    filter_checker(filter_instance).assert_called_once_with(10000)
+                elif filter_type == "limit_negative":
+                    filter_checker(filter_instance).assert_called_once_with(1)
+                else:
+                    filter_checker(filter_instance).assert_called_once_with(filter_value)
+
+    @pytest.mark.parametrize(
+        "case_type,args,error_keyword,assertion_desc",
```

Ruff (0.15.9)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)